account_invoice_amount_adj/models/account_invoice.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    @api.one
+    @api.depends(
+        "invoice_line_ids.price_subtotal",
+        "tax_line_ids.amount",
+        "tax_line_ids.amount_rounding",
+        "currency_id",
+        "company_id",
+        "date_invoice",
+        "type",
+    )
+    def _compute_amount(self):
+        super()._compute_amount()
+        # Use the original logic if a tax other than 
+        # the inclusive tax is applied to account.invoice.line.
```

Comment:
```suggestion
        # Use the original logic for exclusive tax cases.  We assume that that there is no situation where
        # lines contain tax inclusive and exclusive cases in an invoice at the same time.
```
@kanda999 Can you please check if we can apply the same idea (or no need to make adjustments at all) in 15.0?  One concern that I have in mind is the expense scenario where tax exclusive and inclusive items are included in one report (which generates a vendor bill).  We might want to revive your original idea (using the base amount) as necessary.